hashes-generator.php

```diff
@@ -4,42 +4,45 @@
  Author: mattyrob
  Author URI: http://subscribe2.wordpress.com/
 */
-$latest = 'http://wordpress.org/latest.zip';
-$file = 'latest.zip';
+$latest_url = 'http://wordpress.org/latest.zip';
+$zips = array_merge( glob( 'latest.zip' ), glob( 'wordpress-*.zip' ) );
```

Comment:
The best I can come up with here is `glob( 'wordpress-[0-9].[0-9]*.zip' )`

You got anything better?